src/common/copilot/dataverseMetadata.ts

```diff
@@ -57,6 +58,7 @@ export async function getFormXml(entityName: string, formName: string,  orgUrl:
             headers: {
                 'Content-Type': "application/json",
                 Authorization: `Bearer ${apiToken}`,
+                "x-ms-user-agent": "Vscode-Desktop"
```

Comment:
Looks like this header constant is being used in multiple places. Consider moving it's value to a reusable function, with different implementation per Desktop/Web.
e.g. `composeXMsUserAgent`

Reply:
+1 to this comment  @BidishaMS

Reply:
Done